src/error.rs

```diff
@@ -55,3 +57,21 @@ impl From<toml::de::Error> for Error {
         Error::SerdeToml(e)
     }
 }
+
+impl From<parity_wasm::elements::Error> for Error {
+    fn from(error: parity_wasm::elements::Error) -> Self {
+        Error::Cli {
+            message: format!("{}", "There was an output Error"),
```

Comment:
You could just put "There was an output Error".to_string() instead here and below